CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+- Fix mocked module import not working when used as guarded import
```

Comment:
You'll need to fix the merge conflic here and use version 1.18.1

Reply:
Updated